platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/constants/Style.java

```diff
@@ -15,157 +17,33 @@
-     * </p>
-     *
-     * @param version the version of the style.
-     * @return uri to load style from
-     */
-    public static String getMapboxStreetsUrl(int version) {
-        return String.format(MapboxConstants.MAPBOX_LOCALE, MAPBOX_STREETS_BASE, version);
-    }
-
-    /**
-     * Get versioned url of Outdoors streets style.
-     * <ul>
-     * <li>Current version is 9.</li>
-     * </ul>
-     * <p>
-     * More information on the Mapbox styles API can be found on https://www.mapbox.com/api-documentation/#styles
-     * </p>
-     *
-     * @param version the version of the style.
-     * @return uri to load style from
-     */
-    public static String getOutdoorsStyleUrl(int version) {
-        return String.format(MapboxConstants.MAPBOX_LOCALE, OUTDOORS_BASE, version);
-    }
-
-    /**
-     * Get versioned url of Light style.
-     * <ul>
-     * <li>Current default version is 9.</li>
-     * </ul>
-     * <p>
-     * More information on the Mapbox styles API can be found on https://www.mapbox.com/api-documentation/#styles
-     * </p>
-     *
-     * @param version the version of the style.
-     * @return uri to load style from
-     */
-    public static String getLightStyleUrl(int version) {
-        return String.format(MapboxConstants.MAPBOX_LOCALE, LIGHT_BASE, version);
-    }
-
-    /**
-     * Get versioned url of Dark style.
-     * <ul>
-     * <li>Current default version is 9.</li>
-     * </ul>
-     * <p>
-     * More information on the Mapbox styles API can be found on https://www.mapbox.com/api-documentation/#styles
-     * </p>
-     *
-     * @param version the version of the style.
-     * @return uri to load style from
-     */
-    public static String getDarkStyleUrl(int version) {
-        return String.format(MapboxConstants.MAPBOX_LOCALE, DARK_BASE, version);
-    }
-
-    /**
-     * Get versioned url of Satellite style.
-     * <ul>
-     * <li>Current version is 9.</li>
-     * </ul>
-     * <p>
-     * More information on the Mapbox styles API can be found on https://www.mapbox.com/api-documentation/#styles
-     * </p>
-     *
-     * @param version the version of the style.
-     * @return uri to load style from
-     */
-    public static String getSatelliteStyleUrl(int version) {
-        return String.format(MapboxConstants.MAPBOX_LOCALE, SATELLITE_BASE, version);
-    }
 
     /**
-     * Get versioned url of Satellite streets style.
-     * <ul>
-     * <li>Current version is 9.</li>
-     * </ul>
-     * <p>
-     * More information on the Mapbox styles API can be found on https://www.mapbox.com/api-documentation/#styles
-     * </p>
-     *
-     * @param version the version of the style.
-     * @return uri to load style from
+     * Indicates the parameter accepts one of the values from {@link Style}. Using one of these
+     * constants means your map style will always use the latest version and may change as we
+     * improve the style
```

Comment:
I love this documentation.  Is there anyway we could make it bolder though via fonts?  We should strive to make sure there is NO DOUBT that things will change on them.